packages/snaps-cli/src/cmds/init/initHandler.test.ts

```diff
@@ -382,16 +382,7 @@ describe('initialize', () => {
       expect(fsWriteMock).toHaveBeenNthCalledWith(
         4,
         miscUtils.CONFIG_FILE,
-        JSON.stringify(
-          {
-            dist: 'dist',
-            outfileName: 'bundle.js',
-            port: 8081,
-            src: 'src/index.js',
-          },
-          null,
-          2,
-        ),
+        expect.anything(),
```

Comment:
Is there a way to validate this at least somewhat? 🤔

Reply:
Any easy way I thought of, would be re-implementing the implementation of init. I don't want to rely on specific tabs or quoted properties or location of properties in the tests.